src/sysext/sysext.c

```diff
@@ -119,6 +119,24 @@ static const struct {
         }
 };
 
+static int parse_mutable_mode(const char *p) {
+        int r;
+
+        assert(p);
+
+        if (streq(p, "auto"))
+                return MUTABLE_AUTO;
+
+        if (streq(p, "import"))
+                return MUTABLE_IMPORT;
+
+        r = parse_boolean(p);
+        if (r < 0)
+                return r;
+
+        return r ? MUTABLE_YES : MUTABLE_NO;
+}
```

Comment:
we have DEFINE_STRING_TABLE_LOOKUP_WITH_BOOLEAN() to define such "extended boolean" enums